recipe/meta.yaml

```
@@ -42,8 +40,9 @@ build:
   ignore_run_exports:
     - python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --install-option=--use-system-libraries
-  skip: true  # [win or py<36]
+  script:
+    - LIGO_SKYMAP_USE_SYSTEM_CHEALPIX=1 {{ PYTHON }} -m pip install . -vv --install-option=--use-system-libraries
```

```suggestion
    - LIGO_SKYMAP_USE_SYSTEM_CHEALPIX=1 {{ PYTHON }} -m pip install . -vv
```

The `--use-system-libraries` option went away with APE17.